### ISSUE TITLE
Clarify setting up TrustKit in the docs

### ISSUE THREAD

**Issue by myklosbotond** [datatheorem/TrustKit-Android#82]
**Is your feature request related to a problem? Please describe.**

We just ran into the problem described in #21: when the main activity's `onCreate` is called a second time an `IllegalStateException` is thrown. The solution described in the issue works, but I think this can be avoided.

**Describe the solution you'd like**

Make it clearer in the [Getting Started](https://github.com/datatheorem/TrustKit-Android/blob/master/docs/getting-started.md#initializing-trustkit-with-the-pinning-policy) docs in the "Initializing TrustKit with the Pinning Policy" section that the code snippet applies to the `Application`'s `onCreate` method, rather than the `MainActivity`'s method.

**Comment by alexanderdibenedetto** [comment]
Agreed with this, as it was a common enough issue that in the Xamarin version I created I had to add logic to prevent a second initialization attempt.
